app/src/docs/AccountCard/AccountCardPageAdvanced.tsx

```diff
@@ -0,0 +1,258 @@
+/* @license Copyright 2023 @paritytech/polkadot-cloud authors & contributors
```

Comment:
Perhaps instead of having `codeA`, `codeB`, etc, split these examples into separate components? Either in one `AccountCardAdvanced.tsx` or in separate examples within an `Advanced` folder, and so-on, making the examples more readable.